src/main/java/pl/betoncraft/betonquest/objectives/ExperienceObjective.java

```diff
@@ -35,25 +37,39 @@
  */
 public class ExperienceObjective extends Objective implements Listener {
 
-    private final int level;
+    private final int amount;
+    private final boolean level;
```

Comment:
If I read level I'm confused, because I expect it to be the actual level. Maybe rename to `checkForLevel`, `isLevel`, `levelOrPoints` or something else less confusing.